client/src/components/Auth/Login.js

```diff
@@ -3,24 +3,64 @@ import Box from '@mui/material/Box';
 import TextField from '@mui/material/TextField';
 import './Auth.css';
 import Button from "@mui/material/Button";
-import {useDispatch} from "react-redux";
+import {useDispatch, useSelector} from "react-redux";
 import {useState} from "react";
-import {loginServ, logoutServ} from "../../service/auth.service";
-import {logInAction} from "../../store/reducers/loginReducer";
+import {loginServ} from "../../service/auth.service";
+import { ThemeProvider } from '@mui/material/styles';
+import {buttons} from "../muithemes/buttons";
+import InputLabel from "@mui/material/InputLabel";
+import OutlinedInput from "@mui/material/OutlinedInput";
+import InputAdornment from "@mui/material/InputAdornment";
+import IconButton from "@mui/material/IconButton";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import Visibility from "@mui/icons-material/Visibility";
+import FormControl from "@mui/material/FormControl";
 
 export default function Login() {
     const dispatch = useDispatch()
     const logIn = (nick) =>{
-        dispatch({type: "logIn", payload: nick})
+        dispatch(loginServ(nick))
     }
-    const [nickname,setNickname] = useState('')
+    const isLoggedIn = useSelector(state => state.isLoggedIn)
+    if (isLoggedIn){
+        window.location.href = "/search";
```

Comment:
dont use such approach. Read about location navigate and use methods from rounting library